Added instructions on how to configure CaC in the getting started guide (#1449)
OctopusDeploy/docs: docs/shared-content/concepts/create-projects.include.md

1. Navigate to the **Projects** tab, and click **ADD PROJECT**.
1. Give the project a name, for instance, *Hello, world*, and click **Save**.

**Optional**

> By default, Octopus Deploy will store the deployment process, runbook process, and variables in the back-end SQL Server.  Starting with **Octopus 2022.1**, you have the option to store that information in a git repository.  

Reviewer: I think changing this to state upfront what is stored will provide better clarity to customers.

```suggestion
By default, Octopus Deploy will store the deployment process, runbook process, and variables in the back-end SQL Server.  From **Octopus 2022.1**, you have the option to store the *deployment process* in a git repository.  

:::hint
The ability to store runbook processes and variables will be added in future versions of Octopus.
:::
```

Reviewer 2: I agree. I would also add the following.

From **Octopus 2022.1**, you have the option to store the *deployment process* in a git repository, and we're adding support for variables and runbook processes in the future.

> To store that information in source control:

Reviewer: ```suggestion
To configure the project to use version control:
```

Author: Updated

1. Select the option **Use Version Control for this project**
1. Click **Save and Configure VCS**
1. Enter the git repository URL and credentials.
1. Click **Test** to verify the connection.
1. Click **Save** to save the VCS information.

Learn more about [config as code](/docs/projects/version-control/index.md).

> :::hint

Reviewer: ```suggestion
```

> **Please note:** **Octopus 2022.1** added the ability to store the deployment process in a git repo.  The ability to store runbook processes and variables will be added in future versions.

Reviewer: ```suggestion
```

> :::

Reviewer: ```suggestion
```

> :::hint
> **Please note:** **Octopus 2022.1** added the ability to store the deployment process in a git repo.  The ability to store runbook processes and variables will be added in future versions.
> :::

Reviewer: Remove if moved to higher location
```suggestion
```

Author: Moved
